### PR TITLE
Create Models (Version without Migrations)

### PR DESCRIPTION
**Pull request by valeriaDD**

### CODE REVIEW

File: app/Models/Kinetotherapeut.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        'abbreviation',
+    ];
+
+
+    public function  address(){
+
+        return $this->hasOne(Address::class,'kinetotherapist_id');
+    }
+
+    public function appointment(){
```
Comment (vistrati) [inline app/Models/Kinetotherapeut.php:27]: `appointments`

File: app/Models/Day.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class Day extends Model
+{
+    use HasFactory;
+
+    protected $guarded = ['*'];
+
+    public function workingDay(){
+
+        return $this->belongsTo(WorkingDay::class, 'id', 'day_id');
```
Comment (vistrati) [inline app/Models/Day.php:16]: here you should use direct relation `hasOne` and in WorkinDay inverse, please take a look at `Kinetotherapeut` model how you did for address

File: app/Models/Service.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    protected $table = 'services';
+    protected $fillable = [
+        'name',
+        'duration',
+        'price',
+        'abbreviation',
+    ];
+
+    public function appointment(){
```
Comment (vistrati) [inline app/Models/Service.php:20]: `appointments`

File: app/Models/WorkingDay.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+    protected $fillable = [
+        'day_id',
+        'start_time',
+        'end_time'
+    ];
+
+    public function day(){
+
+        return $this->hasMany(Day::class, 'day_id');
```
Comment (vistrati) [inline app/Models/WorkingDay.php:20]: here you should use `belongsTo` and in `Day` has many.

**Comment by vistrati** [comment]
`Schedule` table is missing in your models

**Review by vistrati — CHANGES_REQUESTED** [review]

File: app/Models/Address.php [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    protected $incrementing = false;
+
+    protected $fillable = [
+        'kinetotherapist_id',
+        'city',
+        'street',
+        'street_nr',
+    ];
+
+    protected function kinetotherapeut (){
```
[the remark this answers is not in the document]
Comment (valeriaDD) [reply app/Models/Address.php]: Fixed - made the function public

File: app/Models/Client.php [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    use HasFactory;
+
+    protected $fillable = [
+        'name',
+        'surname',
+        'email',
+        'phone'
+    ];
+
+    public function appointment(){
```
[the remark this answers is not in the document]
Comment (valeriaDD) [reply app/Models/Client.php]: Fixed - changed the name to 'appointments'

File: app/Models/Day.php (hunk 2 above)
Comment (valeriaDD) [reply app/Models/Day.php]: Fixed- made a direct relation between days and working days

File: app/Models/Kinetotherapeut.php (hunk 1 above)
Comment (valeriaDD) [reply app/Models/Kinetotherapeut.php]: Fixed - changed naming to appointments

File: app/Models/Service.php (hunk 3 above)
Comment (valeriaDD) [reply app/Models/Service.php]: Fixed - changed naming to appointments

File: app/Models/WorkingDay.php (hunk 4 above)
Comment (valeriaDD) [reply app/Models/WorkingDay.php]: Fixed - changed this relation to direct one

**Comment by valeriaDD** [comment]
> `Schedule` table is missing in your models



> 

Tried to implement it - but I'm not sure how correct it is - can you take a look one more time?

**Pull request merged**